### DISCUSSION TITLE
Disable seatbelt warning on Dacia Dokker 2016

### DISCUSSION THREAD

**Discussion by KarlBaumann** [cedricp/ddt4all#1266] [General]
Hi!

Is it possible to disable the seatbelt warning on Dacia Dokker 2016 with this tool?

**Comment by DavidIgor** [comment]
easiest is to use macros in pyren !

**Reply by KarlBaumann** [reply]
do you maybe know where to find a tutorial for that? I have never used pyren

**Reply by DavidIgor** [reply]
I doubt that there is tutorial .... it is easy enough
